src/mainIntegrationContext.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const MainIntegrationContext = React.createContext({
+  saveFile: (filePath, data) => {
+    throw new Error(`Couldn't save file at ${filePath} with data: ${data} because `)
+  },
```

Comment:
@Quiescent here, its only throwing an error when the saveFile is called. I didn't get it 😅

Reply:
This is the default behaviour.  It'll only happen when the programmer forgets to give a value to `saveFile` in the context provider.
i.e. if someone forgets to define it in the provider, then that's a mistake and the error should be very visible during development so that we don't accidentally take it into production.
See: https://github.com/cameronsutter/plottr_electron/pull/803/files#diff-bf2d025de96b30a546c2e33beafd0daf68e8b64f2fcc74b6c6c8a135ff6c37abR23